packages/strapi-admin/services/__tests__/user.test.js

```diff
@@ -744,4 +744,90 @@ describe('User', () => {
       );
     });
   });
+
+  describe('resetPasswordByEmail', () => {
+    test('Throws on missing user', async () => {
+      const email = 'email@email.fr';
+      const password = 'invalidpass';
+
+      const findOne = jest.fn(() => {
+        return null;
+      });
+
+      global.strapi = {
+        query() {
+          return {
+            findOne,
+          };
+        },
+      };
+
+      expect.hasAssertions();
+
+      await userService.resetPasswordByEmail(email, password).catch(error => {
```

Comment:
Simple question: Why prefering the use of .`catch(expect)` instead of `expect().toThrow/toThrowError()` ?

Reply:
you must use `expect().rejects.toThrow` but can't do any more expectation in the catch though I think I had more in the beginning will refactor